dotcom-rendering/src/web/components/DynamicSlow.tsx

```diff
@@ -0,0 +1,192 @@
+import { ArticleDesign } from '@guardian/libs';
+
+import { Card } from './Card/Card';
+import { UL } from './Card/components/UL';
+import { LI } from './Card/components/LI';
+
+type Props = {
+	trails: TrailType[];
+};
+
+export const DynamicSlow = ({ trails }: Props) => {
```

Comment:
As a general rule, I think that we should always have some kind of documentation for exported methods. Even if it only contains a link to a Figma file.

```js
/**
 * A `Card` container for up to 8 trails.
 * 
 * [Figma][] · [Chromatic][] · []
 * [Figma]: https://www.figma.com/file/sx2vMFHbL7SsUo0LcpsKNe/%E2%AC%A3--Front-container?node-id=123%3A137122
 * [Chromatic]: https://www.chromatic.com/component?appId=5dfcbf3012392c0020e7140b&csfId=components-dynamicslow&buildNumber=12105&k=62678bafdd54f5004a5dcb68-320-interactive-true&h=6&b=-1
 */
 const DynamicSlow = ({ trails }: Props) => { /* … */ };
```

Reply:
+1

Reply:
Good shout. I added a comment with a link using the [see tag](https://jsdoc.app/tags-see.html).

Note. I removed the link to Chromatic as the issue there is the true, canonical source for that view only exists after a component is merged into main and so either we add these retrospectively or we depend on Chromatic's navigation. That said, if there were some kind of code mod to go through and add these at a later stage that would be cool!